spid_cie_oidc/onboarding/templates/onboarding_landing.html

```diff
@@ -0,0 +1,32 @@
+<!-- Extends default Bootstrap Unical Italia template -->
+{% extends 'base.html' %}
+
+{% load static %}
+{% load i18n %}
+
+{% block page_title %}
+{% trans "OIDC Federation onboarding system" %}
+{% endblock page_title %}
+
+
+{% block container %}
+<div>
+   <div class="container" style="margin-top:20px; padding:1rem">
```

Comment:
Please don't use raw styles with margins/padding

You have to use only the standards bootstrap style classes of bootstrap italia